Sources/AppcuesKit/Presentation/Debugger/ScreenCapturing/UIKitElementTargeting.swift

```diff
@@ -44,6 +47,10 @@ internal class UIKitElementSelector: AppcuesElementSelector {
         // weight the selector property matches by how distinct they are considered
         var weight = 0
 
+        if appcuesID != nil && appcuesID == other.appcuesID {
+            weight += 10_000
```

Comment:
I think using 10000 here is ok? Or do we not want it to match `accessibilityIdentifier`?

Reply:
I was wondering if later we could change that values to constant values that would work with bit operations, Ive never done it but you know like masks, where you can combine masks to have a final value, maybe that would be a way to go here, also the weight values are not as important since when we add new selector types we can always change the weight they have. its not something we need to keep set in stone forever right?